deploy/quickstart/quick_start.py

```diff
@@ -0,0 +1,211 @@
+import os
+from os import path
+import urllib.request
+from subprocess import call
+import subprocess
+
+
+def quick_start():
+    version = "v0.2.0"
+
+    home = os.getenv("HOME")
+    install_dir = request_input_path("Select an installation directory", home + "/jmui")
+
+    bin_dir = install_dir + "/bin"
+    make_or_replace_or_reuse_directory(bin_dir)
+
+    config_dir = install_dir + "/config"
+    make_or_replace_or_reuse_directory(config_dir)
+
+    os.chdir(install_dir)
+
+    service_type = request_input_from_list("Select a shim to use", ["Cromwell", "dsub"], "Cromwell")
+    options = {
+        "Cromwell": quick_start_cromwell
+    }
+    options[service_type](version, install_dir, bin_dir, config_dir)
```

Comment:
typing `dsub` here will raise a `KeyError: 'dsub'` since you are using  `options[service_type]` here, you may want to the call `quick_start_dsub()` here in a `try..except KeyError..` block